src/command/dev/router/config.rs

```diff
@@ -124,15 +125,11 @@ impl RouterConfigHandler {
     }
 
     /// Get the name of the interprocess socket address to communicate with other rover dev sessions
-    pub fn get_ipc_address(&self) -> RoverResult<String> {
-        let socket_name = format!("supergraph-{}.sock", self.get_router_address());
-        {
-            use interprocess::local_socket::NameTypeSupport::{self, *};
-            let socket_prefix = match NameTypeSupport::query() {
-                OnlyPaths | Both => "/tmp/",
-                OnlyNamespaced => "@",
-            };
-            Ok(format!("{}{}", socket_prefix, socket_name))
+    pub fn get_raw_socket_name(&self) -> String {
```

Comment:
Is it worth attempting a test for this change to confirm that the behavior is what we expect? We may have to submit the test as a separate PR and then rebase this branch on top of it to confirm, though.